### PR TITLE
Style seo

### PR DESCRIPTION
**Pull request by veda04**

### CODE REVIEW

**Review by mr-mallik — DISMISSED** [review]
Thanks for the update, Kindly improve the favicons

File: public/index.html [hunk 1]
```diff
@@ -8,8 +8,12 @@
     manifest.json provides metadata used when your web app is added to the
     homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
   -->
-  <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-  <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+  <link rel="icon" type="image/png" href="/favicon/favicon-48x48.png" sizes="48x48" />
```
Comment (mr-mallik) [inline public/index.html]: You need to add %PUBLIC_URL%/ as prefix to these paths

**Review by mr-mallik — CHANGES_REQUESTED** [review]
In index.html you need to make some changes.

File: public/index.html (hunk 1 above)
Comment (veda04) [reply public/index.html]: added %PUBLIC_URL%/ as prefix to these paths #2

**Review by mr-mallik — APPROVED** [review]

**Pull request merged**